framework/platform/bpl/platform_db/prplmesh_platform_db.in

```diff
@@ -0,0 +1,10 @@
+management_mode=@BEEROCKS_MANAGEMENT_MODE@
+operating_mode=@BEEROCKS_OPERATING_MODE@
+stop_on_failure_attempts=@BEEROCKS_STOP_ON_FAILURE_ATTEMPTS@
+certification_mode=@BEEROCKS_CERTIFICATION_MODE@
```

Comment:
Ah, here they are used... Well, I don't think they should be. Management mode and operating mode anyway need to be set by the prplmesh_utils script. The other two you can just hardcode here.

Anyway,nitpick.

Reply:
I prefer to keep data in CMake variables - if config file autogenerated - it should be fully generated.